entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md

```diff
@@ -913,6 +913,34 @@ For example:
 modelBuilder.Entity<Samurai>().HasOne("Some.Entity.Type.Name", null).WithOne();
 ```
 
+## The return type for several async methods has been changed from Task to ValueTask
+
```

Comment:
Please add the link to the tracking issue here (like the other notes)

Reply:
Added.